flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -99,7 +99,7 @@
 
 	private ClosureCleanerLevel closureCleanerLevel = ClosureCleanerLevel.RECURSIVE;
 
-	private int parallelism = PARALLELISM_DEFAULT;
+	private int parallelism = CoreOptions.DEFAULT_PARALLELISM.defaultValue();
```

Comment:
@dawidwys this changes the default parallelism from -1(undefined) to 1, is this what we expect?